Gemfile

```diff
@@ -114,6 +112,22 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-c3', '~> 0.4.10'
   gem 'rails-assets-bootstrap-select', '~> 1.5.4'
   gem 'rails-assets-bootstrap-hover-dropdown', '~> 2.0.11'
+
+  gem 'rails-assets-angular', '~>1.4.3'
+  gem 'rails-assets-angular-animate', '~> 1.4.3'
+  gem 'rails-assets-angular-aria', '~> 1.4.3'
+  gem 'rails-assets-angular-cookies', '~> 1.4.3'
+  gem 'rails-assets-angular-loader', '~> 1.4.3'
+  gem 'rails-assets-angular-message-format', '~> 1.4.3'
+  gem 'rails-assets-angular-messages', '~> 1.4.3'
+  gem 'rails-assets-angular-mocks', '~> 1.4.3'
+  gem 'rails-assets-angular-resource', '~> 1.4.3'
+  gem 'rails-assets-angular-route', '~> 1.4.3'
+  gem 'rails-assets-angular-sanitize', '~> 1.4.3'
+  gem 'rails-assets-angular-scenario', '~> 1.4.3'
+  gem 'rails-assets-angular-touch', '~> 1.4.3'
```

Comment:
Not really, it's just all the stuff angularjs-rails adds. It's just that a quick search didn't show me mocks which I *know* we use, so I want to go through them properly to be sure